client/src/components/FormField/RegisteredMFAMethodListField/RegisteredMFAMethodListField.scss

```diff
@@ -1,43 +1,72 @@
 .registered-mfa-method-list-field {
+  // Make this a sort of "bootstrap context"
+  @import '~bootstrap/scss/reboot';
+  @import '~bootstrap/scss/buttons';
+
   padding: $input-padding-y 0;
 
-  &__button,
-  &__no-methods {
+  // We need extra specificity here because we've got "reboot" inside this context.
+  & &__button,
+  & &__no-methods {
     margin-bottom: $input-padding-y-lg;
   }
-}
 
-.method-list {
-  list-style: none;
-  margin: 0;
-  padding: 0;
+  .method-list {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  .registered-method-list-item {
+    margin-bottom: $input-padding-y-lg;
+
+    &__control {
+      border: 0;
+      margin: 0;
+      padding: 0;
+      background: none;
+      text-decoration: none;
+      color: $link-color;
+      cursor: pointer;
+
+      &:hover {
+        color: $link-hover-color;
+        text-decoration: underline;
+      }
+    }
+  }
 }
 
-.registered-method-list-item {
-  margin-bottom: $input-padding-y-lg;
+.modal-dialog {
+  // Make this a sort of "bootstrap context"
+  @import '~bootstrap/scss/reboot';
+  @import '~bootstrap/scss/buttons';
 
-  &__control {
-    border: 0;
-    margin: 0;
-    padding: 0;
-    background: none;
-    text-decoration: none;
-    color: $link-color;
-    cursor: pointer;
-
-    &:hover {
-      color: $link-hover-color;
-      text-decoration: underline;
+  // Various cms overrides to make bootstrap modals look nice in SS3
+  .modal-title {
```

Comment:
these classnames seem kinda generic - I realise they're dictated by bootstrap, but have you double checked for conflicts, just in case?

Reply:
I have. The existing modals don't share these classes. If they did there is nothing I could do about it short of implementing our own modal components instead of using reactstraps ones.

Reply:
Neat, thanks for confirming :)
I was thinking a wrapper 'namespace' class would be the only way (i.e. introduce specificity via a new ancestor class).
Good to know that's not needed.

Reply:
Yeah unfortunately the modals are appended to the body - so an ancestor class is not possible :(